code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubClient.java

```diff
@@ -104,16 +106,42 @@ public GithubAccountInfo getUserAccountInfo(String token) throws IOException {
         return orgInfoList;
     }
 
-    public GithubAccountInfo getUserIdByName(String username) throws IOException {
+    public List<TeamAccountInfo> getOrgTeamInfo(String token, String org) throws IOException {
+        if (StringUtils.isEmpty(token)) {
+            return null;
+        }
+        List<TeamAccountInfo> teamInfoList = new ArrayList<>();
+        List<Map<String, Object>> jsonData = null;
+
+        HttpResponse response = Request.Get(GITHUB_UNAUTH_ORG_URL.get() + org + "/teams").addHeader("Authorization", "token " + token)
+                .addHeader("Accept", "application/json").execute().returnResponse();
+        int statusCode = response.getStatusLine().getStatusCode();
+        if (statusCode != 200) {
+            throw new ClientVisibleException(ResponseCodes.SERVICE_UNAVAILABLE);
+        }
+        jsonData = jsonMapper.readCollectionValue(response.getEntity().getContent(), List.class, Map.class);
+
+        for (Map<String, Object> orgObject : jsonData) {
+            String accountId = ObjectUtils.toString(orgObject.get("id"));
+            String accountName = ObjectUtils.toString(orgObject.get("name"));
+            if (!StringUtils.equals("Owners", accountName)) {
```

Comment:
This isn't a http header, but i'm guessing you should probably do case insensitive check here.